src/configure

```diff
@@ -896,6 +945,41 @@ fi
 # Most of the OS-specific steps below will append to kaldi.mk
 echo "Doing OS specific configurations ..."
 
+if $android ; then
+  OPENFSTLIBS="$FSTROOT/lib/libfst.a"
+  echo "OPENFSTLIBS = $OPENFSTLIBS" >> kaldi.mk
+
+  if [ -z $TOOLCHAININC ] ;  then
+    failure "--toolchain-includes must be specified for android builds"
+  fi
+
+  if [ -z $HOST ] ; then
```

Comment:
What happens if we are cross compiling for a platform other than android and host is not set?